### PR TITLE
Issue 143 ✨ feature add landing page in light mode

### LINKED ISSUE #143: ✨ Feature: Add landing page in light mode
# ℹ Overview

Add Steph's designs into the next.js project and make it the new ./pages/index.js. This does **not** need to use MUI components since it's just a landing page.

https://www.figma.com/file/3mUNzhDN7whCLAMlykP8UY/Mailbox-Landing-Page?node-id=0%3A1

[MailboxPageLight2.pdf](https://github.com/algodex/algodex-mailbox/files/8413136/MailboxPageLight2.pdf)

"Hope everyone is having a good Monday! :sunny:
I've been working on a landing page for Algodex Mailbox. It's simple with a brief overview of the tool and directs to the App, User Guide, and FAQ. We can add future sections with stats and projects that have used it for airdrops for example. I also just used a few FAQs - we can decide which ones are needed here (This section is also expandable so it's not just a ton of text). Let me know any feedback! There's a light and dark version but they are the same otherwise" -Steph

### PR DESCRIPTION
**Pull request by treasurechic**
# ℹ Overview

### 📝 Related Issues

-- closes #143

### CODE REVIEW

File: components/LandingPage/Hero.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                  alt="Powered by Algorand"
+                  width="150"
+                  height="20"
+                />
+                <p>
+                  Algodex Mailbox is a decentralized web application that allows
+                  users to send Algorand Standard Assets even if recipients
+                  haven’t opted into the asset yet. <br /> <br />
+                  Useful when sending to one wallet or thousands. <br /> <br />
+                  <strong>Try on testnet Now:</strong>
```
Comment (atrefonas) [inline components/LandingPage/Hero.js]: Use the /lib function to check what network you are on, and change to Mainnet or Testnet depending on this.

**Review by atrefonas — CHANGES_REQUESTED** [review]
image


This picture is missing

File: components/LandingPage/Hero.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                  src="/Powered-by-Algorand.png"
+                  alt="Powered by Algorand"
+                  width="150"
+                  height="20"
+                />
+                <p>
+                  Algodex Mailbox is a decentralized web application that allows
+                  users to send Algorand Standard Assets even if recipients
+                  haven’t opted into the asset yet. <br /> <br />
+                  Useful when sending to one wallet or thousands. <br /> <br />
```
Comment (atrefonas) [inline components/LandingPage/Hero.js]: Change from 'thousands' to 'hundreds' for now.

**Comment by atrefonas** [comment]
Questions 2 and 3 are the same.

The answers to all 3 are the same. Please fix

image

**Comment by atrefonas** [comment]
image


The logo is missing

**Comment by atrefonas** [comment]
"Learn to use Algodex Mailbox using our guide."

Needs a link to the guide

**Comment by atrefonas** [comment]
Can you reduce the spacing between the unexpanded boxes? To be the same as the spacing between the expanded boxes.

image

**Comment by atrefonas** [comment]
The colors are wrong in the nav bar. It should look like this:

image

**Comment by treasurechic** [comment]
No, the colors are not wrong, https://www.figma.com/file/3mUNzhDN7whCLAMlykP8UY/Mailbox-Landing-Page?node-id=36%3A2

I had a discussion with  Steph on the colors, told her it will not make sense to have a different color on the landing page and have a different one on the dashboard. So she created the theme for purple which is the existing color palette. 

I can change the color to the grey theme if that is what you want, but that means we have to change the whole theme to grey so we can have a unigue color code with the dashboard

**Comment by atrefonas** [comment]
Your box is too small, the fonts are too small, and should also be under part of the graphic design.

Design:

image



Yours:

image

**Comment by treasurechic** [comment]
Oh, large screens...will update

**Comment by atrefonas** [comment]
Yes, please make sure it looks good on every screen size.

**Comment by atrefonas** [comment]
It doesn't work in Spanish mode. Either disable Spanish or provide translations.

https://algodex-mailbox-mo3yq2i08-algodex-dev.vercel.app/es#user-guide

**Comment by treasurechic** [comment]
It works for spanish, I just can't translate the whole page, that is why it is not reflecting on the whole page(I was hoping the language translator will do that).

Here is an example of where it works currently
image

**Comment by atrefonas** [comment]
Can you use Google Translate to translate the whole page?

**Comment by stephclarkga** [comment]
I'm not against changing the whole theme to gray if everyone agrees since I think it ties into the original Algodex colors better but I did update this to match the colors Adesola provided were already in use. 

Couple things that haven't been mentioned yet: 

This third box ended up a bit bigger than the others - due to padding around the text maybe? Text should be moved up a bit so the box is the same as the others
Screen Shot 2022-04-08 at 3 33 08 PM

"Mailbox User Guide" should link to https://about.algodex.com/docs/algodex-mailbox-user-guide/
And only "Mailbox User Guide" should be underlined to show it's a link.
Screen Shot 2022-04-08 at 3 30 21 PM

I think we can remove the "Careers" and "Partnerships" from the footer right now until we have pages they can link to. We could add a link to Algonaut's About page under Company to tie that in if Alex thinks so or just remove that whole "Company" section. Also where did the email list join section go? It was there when I looked at the page earlier - maybe the expanded FAQ is covering it now?
Screen Shot 2022-04-08 at 3 36 51 PM

**Comment by treasurechic** [comment]
@stephclarkga , the email input is hidden for now until it is functional

**Comment by stephclarkga** [comment]
Also, in the header, let's have "User Guide" link to https://about.algodex.com/docs/algodex-mailbox-user-guide/

And in the FAQ, can you make the questions bold like this? It's easier to read when the question text is different than the answer
Screen Shot 2022-04-08 at 3 46 23 PM

**Comment by treasurechic** [comment]
Everything is noted, I will adjust.

**Comment by stephclarkga** [comment]
Awesome, it looks great just those few minor things!

**Review by stephclarkga — APPROVED** [review]

**Pull request merged**